src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -129,14 +129,15 @@ public void assertEquals(int expectedCode) {
      * @param expectedJson - the expected response json object
      */
     public void assertEquals(JsonObject expectedJson) {
-        Success success = Success.FAIL;
+        boolean pass = false;
         if (object != null) {
-            success = object.equals(expectedJson) ? Success.PASS : Success.FAIL;
+            pass = object.equals(expectedJson) ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1125)